source/tutorials/working-with-local-files.md

```diff
@@ -366,17 +366,31 @@ One way to fix this is to use [`unions`](https://nixos.org/manual/nixpkgs/stable
 
 Create a file set containing a union of the files to exclude (`fs.unions [ ... ]`), and subtract it (`difference`) from the complete directory (`./.`):
 
-```{code-block} nix
+```{code-block} diff
```

Comment:
We don't always have to make it a difference. There's no firm rule, but I tried making it a diff where it's a small change - in order to highlight that - and have a full file, like here, where it's essentially a new thing.

Reply:
[11c9f26](https://github.com/NixOS/nix.dev/pull/1025/commits/11c9f2643881fde0e4446d53a1493822434a8529)
Ok, I remove code defferences and change the description of PR.